src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/PackageNameCheck.java

```diff
@@ -71,14 +71,14 @@ public class PackageNameCheck
      */
     public static final String MSG_KEY = "name.invalidPattern";
 
-    /** Specifies valid identifiers. */
+    /** Sets the pattern to match valid identifiers. */
     // Uppercase letters seem rather uncommon, but they're allowed in
     // https://docs.oracle.com/javase/specs/
     //  second_edition/html/packages.doc.html#40169
     private Pattern format = Pattern.compile("^[a-z]+(\\.[a-zA-Z_]\\w*)*$");
 
     /**
-     * Setter to specifies valid identifiers.
+     * Setter to sets the pattern to match valid identifiers.
```

Comment:
this a bit weird, but it was a demand from xdoc-javadoc validation test. This module has this property not from abstract class, I tried it be at least be worded in same way. But it was not easy to reconcile all.

Reply:
Are we able to at least do `sets` -> `set`? `sets` in not correct grammar in this context.

Reply:
reworded.